.github/workflows/ci_lin.yml

```diff
@@ -60,10 +60,11 @@ jobs:
           cd c2s.src
           mkdir build
           cd build
+          export LLVM_SYMBOLIZER_PATH=$(pwd)/bin/llvm-symbolizer
           cmake \
             -G Ninja \
             -DLLVM_ENABLE_PROJECTS="clang"  \
-            -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_BUILD_TYPE=RelWithDebInfo \
```

Comment:
```suggestion
            -DCMAKE_BUILD_TYPE=Debug \
```

Reply:
Please use **Debug** build to avoid strange backtrace due to compiler optimization.